documentation/modules/auxiliary/client/sms/send_text.md

```diff
@@ -0,0 +1,132 @@
+The ```auxiliary/client/sms/send_text``` module allows you to send a malicious text/link to a collection
+of phone numbers of the same carrier.
+
+In order to use this module, you must set up your own SMTP server to deliver messages. Popular
+mail services such as Gmail, Yahoo, Live should work fine.
+
+## Module Options
+
+**CELLNUMBERS**
+
+The 10-digit phone number (or numbers) you want to send the text to. If you wish to target against
+multiple phone numbers, ideally you want to create the list in a text file (one number per line),
+and then load the CELLNUMBERS option like this:
+
+```
+set CELLNUMBER file:///tmp/att_phone_numbers.txt
```

Comment:
Just a little typo here @wchen-r7 . The above line should be:
```ruby
set CELLNUMBERS file:///tmp/att_phone_numbers.txt
```
s/CELLNUMBER/CELLNUMBERS/g